test/core/security/credentials_test.cc

```diff
@@ -449,22 +449,39 @@ static void test_oauth2_token_fetcher_creds_parsing_missing_token_lifetime(
   grpc_http_response_destroy(&response);
 }
 
-typedef struct {
+namespace {
+
+struct ExpectedMetadata {
   const char* key;
   const char* value;
-} expected_md;
+};
+
+struct RequestMetadataState {
```

Comment:
Let's turn this into a proper class (make data members private and change static functions that act on it to be methods).

Reply:
Done.